main.go

```diff
@@ -22,6 +22,9 @@ import (
 	"syscall"
 	"time"
 
+	metricslegacy "github.com/observatorium/api/api/metrics/legacy"
```

Comment:
Please maintain the imports ordered following the golang convention of grouping first standard library imports, then third party imports, and lastly imports from our module.